frontend/src/scenes/data-warehouse/editor/OutputPane.tsx

```diff
@@ -136,11 +94,34 @@ export function OutputPane(): JSX.Element {
 
     const vizKey = useMemo(() => `SQLEditorScene`, [])
 
+    const [selectedRow, setSelectedRow] = useState<Record<string, any> | null>(null)
+
+    const setProgress = useCallback((loadId: string, progress: number) => {
+        setProgressCache((prev) => ({ ...prev, [loadId]: progress }))
+    }, [])
+
     const columns = useMemo(() => {
         const types = response?.types
 
-        return (
-            response?.columns?.map((column: string, index: number) => {
+        const baseColumns = [
+            {
+                key: '__details',
+                name: '',
+                width: 15,
+                renderCell: ({ row }: { row: any }) => (
+                    <div className="hover-actions-cell flex justify-center items-center">
+                        <LemonButton
+                            size="xsmall"
+                            icon={<IconExpand45 />}
+                            onClick={(e) => {
+                                e.stopPropagation()
+                                setSelectedRow(row)
+                            }}
+                        />
+                    </div>
+                ),
+            },
```

Comment:
I see that we're taking some inspiration from metabase.... It looks like the keyboard selection navigates to the "empty" column. Is there an easy way to disable this with the framework?
<img width="1076" alt="image" src="https://github.com/user-attachments/assets/4af2289a-d433-48f0-8e6e-c117c0cf683a" />